app/src/main/java/com/fstyle/androidtrainning/MainActivity.java

```diff
@@ -1,13 +1,72 @@
 package com.fstyle.androidtrainning;
 
+import android.support.annotation.IntDef;
+import android.support.v4.view.ViewPager;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
-public class MainActivity extends AppCompatActivity {
+import com.fstyle.androidtrainning.screen.ViewPagerAdapter;
+import com.roughike.bottombar.BottomBar;
+import com.roughike.bottombar.OnTabSelectListener;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import static com.fstyle.androidtrainning.MainActivity.Tab.FAVORITE;
+import static com.fstyle.androidtrainning.MainActivity.Tab.HOME;
+import static com.fstyle.androidtrainning.MainActivity.Tab.PROFILE;
+import static com.fstyle.androidtrainning.MainActivity.Tab.SEARCH;
+
+public class MainActivity extends AppCompatActivity implements OnTabSelectListener {
+    private ViewPager mViewPager;
+    private BottomBar mBottombar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        initViews();
+        final ViewPagerAdapter viewPagerAdapter = new ViewPagerAdapter(getSupportFragmentManager());
+        mViewPager.setAdapter(viewPagerAdapter);
+        handleEvents();
+    }
+
+    private void initViews() {
+        mViewPager = findViewById(R.id.unSwipViewPager);
+        mBottombar = findViewById(R.id.bottomBar);
+    }
+
+    private void handleEvents() {
+        mBottombar.setOnTabSelectListener(this);
+    }
+
+    @Override
+    public void onTabSelected(int tabId) {
+        switch (tabId) {
+            case R.id.tabhome:
+                mViewPager.setCurrentItem(Tab.HOME);
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'Tab.HOME' due to existing static import 'com.fstyle.androidtrainning.MainActivity.Tab.HOME'
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName